<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Micmac selection tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icones/icones.qrc">
    <normaloff>:/MM/Icons/images/icon_32.png</normaloff>:/MM/Icons/images/icon_32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(200,200,200);
}

QMenu
{
	background: rgb(200,200,200);
}

QMenu::separator 
{
     height: 2px;
     background: rgb(100,100,100);
     margin-left: 10px;
     margin-right: 5px;
 }

QMenuBar 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(255,255,255), stop:1 rgb(200,200,200));
 }

QStatusBar
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(200,200,200), stop:1 rgb(255,255,255));
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="OpenglLayout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border: 2px solid white;
	border-radius: 10px;
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(10,102,151), stop:1 rgb(0,0,0));
 }

</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuStandard_views">
     <property name="title">
      <string>Standard views</string>
     </property>
     <addaction name="actionSetViewTop"/>
     <addaction name="actionSetViewFront"/>
     <addaction name="actionSetViewLeft"/>
     <addaction name="actionSetViewBack"/>
     <addaction name="actionSetViewRight"/>
     <addaction name="actionSetViewBottom"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuStandard_views"/>
    <addaction name="actionFullScreen"/>
    <addaction name="separator"/>
    <addaction name="actionTogglePoints_selection"/>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHelpShortcuts"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionHelpShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
   <property name="toolTip">
    <string>Shortcuts</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewFront">
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewYpos.png</normaloff>:/images/ccViewYpos.png</iconset>
   </property>
   <property name="text">
    <string>Front</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewBack">
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewYneg.png</normaloff>:/images/ccViewYneg.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewLeft">
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewXpos.png</normaloff>:/images/ccViewXpos.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewRight">
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewXneg.png</normaloff>:/images/ccViewXneg.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewTop">
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewZpos.png</normaloff>:/images/ccViewZpos.png</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSetViewBottom">
   <property name="icon">
    <iconset>
     <normaloff>:/images/ccViewZneg.png</normaloff>:/images/ccViewZneg.png</iconset>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionTogglePoints_selection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points selection</string>
   </property>
   <property name="toolTip">
    <string>Toggle between Points selection mode and standard mode</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Undo selection</string>
   </property>
   <property name="toolTip">
    <string>Undo all past selections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icones/icones.qrc"/>
 </resources>
 <connections/>
</ui>
